x-pack/plugins/infra/public/containers/metrics_source/source_errors.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint-disable max-classes-per-file */
+
+import { HttpFetchError } from '@kbn/core-http-browser-internal/src/http_fetch_error';
+import { i18n } from '@kbn/i18n';
+
+const NO_SUCH_REMOTE_CLUSTER = 'no_such_remote_cluster_exception';
+
+const defaultTitle = i18n.translate('xpack.infra.sourceConfiguration.failedToFetchSource', {
+  defaultMessage: 'Failed source request.',
+});
+
+const noSuchRemoteClusterMessage = i18n.translate(
+  'xpack.infra.sourceConfiguration.noSuchRemoteCluster',
+  {
+    defaultMessage:
+      'The remote cluster is not configured or not available.\nCheck the metric index settings and make sure they are written correctly.',
```

Comment:
```suggestion
      'Some remote cluster is not configured or is unavailable.\nCheck the metric source index setting and make sure they are configured correctly.',
```

Reply:
Totally, @formgeist @kkurstak any suggestions? Or should we stay with Milton's suggestion?
```
Some remote cluster is not configured or is unavailable.\nCheck the metric source index setting and make sure they are configured correctly.
```

Reply:
"Some" sounds a bit awkward. 

I think:
`A remote cluster has not been configured or is unavailable. Check the metrics indices value under Settings (link to settings).`

Maybe we can a link to CCS or remote clusters docs too.

Reply:
Sounds good, we can add a "Learn more" link that brings to the remote clusters docs 👍

Reply:
I wrote "Some" since we don't know if it's one or many 🤷🏼